tracer/src/Datadog.Trace/AppSec/Waf/Context.cs

```diff
@@ -66,32 +66,26 @@ private Context(IntPtr contextHandle, IWaf waf, IWafLibraryInvoker wafLibraryInv
     public Dictionary<string, object> ShouldRunWith(IDatadogSecurity security, string? userId = null, string? userLogin = null, string? userSessionId = null, bool fromSdk = false)
```

Comment:
NIT: Maybe it's a little confusing having these two methods with the same name, very similar signature, and different return values?

Dictionary<string, object> ShouldRunWith(IDatadogSecurity security, string? userId = null, string? userLogin = null, string? userSessionId = null, bool fromSdk = false)

private bool ShouldRunWith(IDatadogSecurity security, UserEventsState.UserRecord? userRecord, string? value, string address, bool fromSdk)

Reply:
Good point, changed and documented in [061a364](https://github.com/DataDog/dd-trace-dotnet/pull/6623/commits/061a3649c233ef0ac89c743809db0fdd3c9df362)